feature/home/src/main/java/org/sopt/home/HomeScreen.kt

```diff
@@ -0,0 +1,184 @@
+package org.sopt.home
+
+import androidx.compose.animation.core.LinearOutSlowInEasing
+import androidx.compose.animation.core.tween
+import androidx.compose.foundation.ExperimentalFoundationApi
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.LazyListState
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.lazy.rememberLazyListState
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Add
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.FloatingActionButton
+import androidx.compose.material3.Icon
+import androidx.compose.material3.SheetState
+import androidx.compose.material3.rememberModalBottomSheetState
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color.Companion.White
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import org.orbitmvi.orbit.compose.collectAsState
+import org.sopt.designsystem.component.dialog.SoptAlertDialog
+import org.sopt.designsystem.component.textfield.RegularTextField
+import org.sopt.designsystem.ui.theme.NOWSOPTAndroidTheme
+import org.sopt.model.Friend
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun HomeRoute(
+    viewModel: HomeViewModel = hiltViewModel(),
+    label: String,
+) {
+    val state = viewModel.collectAsState().value
```

Comment:
- value를 사용해서 구독하는 것보다 by(getValue) 사용해보는 것이 가독성에 더 좋을 것 같습니다.
- 아래와 같이 state 구독함수를 변경하면 어떤 점이 달라질까요?

```suggestion
    val state by viewModel.collectAsStateWithLifecycle()
```

Reply:
composable의 lifecycle에 맞게 데이터 수집을 관리해주기 때문에 보다 효율적일 것 같습니다. 감사합니다 !!